protocol-units/execution/maptos/framework/releases/README.md

```diff
@@ -13,4 +13,37 @@ We use this directory to track framework releases and migrations. The path of mi
     4. Implement the new release in the new directory.
     5. Write a migration test for it against the named release which is now in the [`parent`](./parent/) directory.
     6. Update the [`latest`](./latest/) directory to re-export the new release.
-- The [`head`](./head/) crate is a special case that is used to track the release that is on the current branch head. It can be used for intermediately testing changes that are not yet ready to be published.
\ No newline at end of file
+- The [`head`](./head/) crate is a special case that is used to track the release that is on the current branch head. It can be used for intermediately testing changes that are not yet ready to be published.
+
+# Feature migration
+
+The feature comparission between MVT Network and Aptos Network gives these features that are different.
+
+| Title                                      | MVT     | APTOS   | MIGRATED Network |
```

Comment:
this column `MIGRATED Network` mean we want to do the feature change after the upgrade?  would it be easier to do it now on the movement mainnet since we have core-resource signer. After migration, it requires governance proposal to change the feature flag

Reply:
Yes, we'll do the change when it's the best. This column doesn't indicate when to do the change but only the state after the migration. After this PR, we need to determine for each feature when it's the best to change them during the migration process.